fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs

```diff
@@ -220,23 +222,21 @@
     .AddPolicy<string, SampleRateLimiterPolicy>(helloPolicy)
     .AddPolicy<string>(userPolicyName, context =>
     {
-        if (context.User?.Identity?.IsAuthenticated is not true)
+        var username = "anonymous user";
+        if (context.User?.Identity?.IsAuthenticated is true)
         {
-            var username = "anonymous user";
-
-            return RateLimitPartition.CreateSlidingWindowLimiter<string>(username,
-                  key => new SlidingWindowRateLimiterOptions(
-                  permitLimit: myOptions.permitLimit,
-                  queueProcessingOrder: QueueProcessingOrder.OldestFirst,
-                  queueLimit: myOptions.queueLimit,
-                  window: TimeSpan.FromSeconds(myOptions.window),
-                  segmentsPerWindow: myOptions.segmentsPerWindow
-                ));
-        }
-        else
-        {
-            return RateLimitPartition.CreateNoLimiter<string>(string.Empty);
```

Comment:
@Kahbazi removing `CreateNoLimiter` since it's gone in RC1.

Reply:
@Rick-Anderson I think `AddNoLimiter` is removed in ASP.NET Core in RC1. We still have `CreateNoLimiter` in runtime, and it should be used in this sample. Right now one anonymous user could block the api for all other anonymous users.